tests/meson.build

```
@@ -0,0 +1,64 @@
+tests_env = environment()
+tests_env.prepend('GI_TYPELIB_PATH', meson.project_build_root() / 'eosmetrics')
+
+tests_c_args = [
+  '-DG_LOG_DOMAIN="EosMetrics"',
+  '-DCOMPILING_EOS_METRICS',
+  '-D_POSIX_C_SOURCE=200112L',
+]
+
+
+###########################
+# test-daemon-integration #
+###########################
+
+python = import('python')
+python_exe = python.find_installation('python3')
```

```suggestion
python_exe = find_program('python3')
```

You're not using this for the primary purpose of the python module, which is to build and install python software / python C extensions. For which purpose, the find_installation method always runs an introspection script in the detected python, to sanity-check it, return an error if distutils isn't installed (this is a problem by default on Debian), and then produce fairly extensive json reporting output for all the little knobs available to distutils/sysconfig with regard to building C extensions.